utils/namespace.py

```diff
@@ -0,0 +1,3 @@
+class Namespace:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
based on proposed changes [here](https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/239/files#r308269996) I would drop this file from the PR.

Reply:
done